packages/cli/test/unit/commands/git/index.test.ts

```diff
@@ -21,9 +27,22 @@ describe.todo('git', () => {
     });
   });
 
-  it('errors when invoked without subcommand', async () => {
-    client.setArgv('git');
-    const exitCodePromise = git(client);
-    await expect(exitCodePromise).resolves.toBe(2);
+  it('displays help when invoked without subcommand', async () => {
+    const cwd = fixture('new-connection');
```

Comment:
Honestly not a great sign all of this is a precondition to saying "you need to supply a subcommand"